service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseRoleService.java

```diff
@@ -92,9 +93,15 @@ private void grantCaseAssignmentAccess(String caseId, Set<String> users, Set<Cas
         try {
             final String userToken = idam.getAccessToken(userConfig.getUserName(), userConfig.getPassword());
             final String serviceToken = authTokenGenerator.generate();
+
+            final String organisationId = organisationService.findOrganisation()
+                .map(Organisation::getOrganisationIdentifier)
+                .orElse(null);
+
             List<CaseAssignedUserRoleWithOrganisation> caseAssignedRoles = users.stream()
                 .map(user -> CaseAssignedUserRoleWithOrganisation.builder()
                     .caseDataId(caseId)
+                    .organisationId(organisationId)
```

Comment:
as per agreement with CCD : RDM-9557
No organisation ID field in your requests - Org ID fields will be incorporated later on with agreement with CCD,

Reply:
Do we need to inform CCD before deploying this change?

Reply:
thanks for pointing that, contacted with Mutlu to confirm if we can use it

Reply:
had chat with Mutlu and Sam about this one, we have green light to